src/PlaywrightRunner.ts

```diff
@@ -120,8 +125,11 @@ class PlaywrightRunner extends JestRunner {
     onFailure: OnTestFailure,
     options: TestRunnerOptions,
   ): Promise<void> {
-    const browserTests = await this.getTests(tests)
-
+    const config = await readConfig(tests[0].context.config.rootDir)
```

Comment:
I'm not sure that using `rootDir` from first test is a good decision

Reply:
We could also use the global rootDir, do you think this is better? Also not sure tho

Reply:
> We could also use the global rootDir, do you think this is better? Also not sure tho

global rootDir won't work :( https://github.com/playwright-community/jest-playwright/issues/120

Reply:
Then we have to make it either per test which would be super complicated or let it as it is. Should be probably for most use cases or what do you think?

Reply:
@mxschmitt I think we can test it out for a while. If there will be some issues related to `rootDir` we can fix it

Reply:
so whats better, the global rootDir or the rootDir of the first test?

Reply:
> so whats better, the global rootDir or the rootDir of the first test?

the rootDir of the first test, cause global rootDir is not correct to use for this